### PR TITLE
[Design] [MG] Module Decomposition

### LINKED ISSUE #352: [Design] [MG] Module Decomposition - User Service
## Content

- Module Decomposition

## Rubric Sections

- SoftArchitec: Quality Information
- SoftArchitec: Presentation

### LINKED ISSUE #353: [Design] [MG] Module Decomposition - Shop Appointments
## Content

- Module Decomposition

## Rubric Sections

- SoftArchitec: Quality Information
- SoftArchitec: Presentation

### LINKED ISSUE #354: [Design] [MG] Module Decomposition - Customer Appointments
## Content

- Module Decomposition

## Rubric Sections

- SoftArchitec: Quality Information
- SoftArchitec: Presentation

### LINKED ISSUE #355: [Design] [MG] Module Decomposition - Quotes
## Content

- Module Decomposition

## Rubric Sections

- SoftArchitec: Quality Information
- SoftArchitec: Presentation

### LINKED ISSUE #356: [Design] [MG] Module Decomposition - Work Orders
## Content

- Module Decomposition

## Rubric Sections

- SoftArchitec: Quality Information
- SoftArchitec: Presentation

### LINKED ISSUE #357: [Design] [MG] Module Decomposition - Services
## Content

- Module Decomposition

## Rubric Sections

- SoftArchitec: Quality Information
- SoftArchitec: Presentation

### LINKED ISSUE #358: [Design] [MG] Module Decomposition - Shop Lookup
## Content

- Module Decomposition

## Rubric Sections

- SoftArchitec: Quality Information
- SoftArchitec: Presentation

### LINKED ISSUE #359: [Design] [MG] Module Decomposition - Shop Profile
## Content

- Module Decomposition

## Rubric Sections

- SoftArchitec: Quality Information
- SoftArchitec: Presentation

### PR DESCRIPTION
**Pull request by arkinmodi**
Close https://github.com/arkinmodi/project-sayyara/issues/352
Close https://github.com/arkinmodi/project-sayyara/issues/353
Close https://github.com/arkinmodi/project-sayyara/issues/354
Close https://github.com/arkinmodi/project-sayyara/issues/355
Close https://github.com/arkinmodi/project-sayyara/issues/356
Close https://github.com/arkinmodi/project-sayyara/issues/357
Close https://github.com/arkinmodi/project-sayyara/issues/358
Close https://github.com/arkinmodi/project-sayyara/issues/359

### CODE REVIEW

File: docs/Design/SoftArchitecture/MG.tex [hunk 1]
```diff
@@ -257,40 +259,121 @@ \subsection{Hardware Hiding Modules (\ref{mHH})}
...
+	\item[Implemented By:] Tiny Coders
+		% \item[Type of Module:]
+\end{description}
+
+\subsubsection{Quotes Module (\ref{mQuotes})}
+
+\begin{description}
+	\item[Secrets:] The format and structure of quote and chat message data objects.
+	\item[Services:] Provide CRUD operations for quote and chat message objects.
+	\item[Implemented By:] Tiny Coders
```
Comment (TimChoy) [inline docs/Design/SoftArchitecture/MG.tex]: Should `Implemented by` be our team name or our program name?

File: docs/Design/SoftArchitecture/MG.tex [hunk 2]
```diff
@@ -257,40 +259,121 @@ \subsection{Hardware Hiding Modules (\ref{mHH})}
...
+%              for leaf modules in the decomposition by secrets tree.]
+% \end{description}
+
+\subsubsection{User Module (\ref{mUsers})}
+
+\begin{description}
+	\item[Secrets:] The format and structure of user data objects.
+	\item[Services:] Provide CRUD operations for user objects, and the ability to verify a user's
+		authorization.
+	\item[Implemented By:] Tiny Coders
```
Comment (joyxiao99) [inline docs/Design/SoftArchitecture/MG.tex]: I think we should put our actual name here like who’s working on it. Thats how we did it in the other course anyways

Comment (arkinmodi) [reply docs/Design/SoftArchitecture/MG.tex]: The example the prof had says

```latex
\item[Implemented By:] [Your Program Name Here]
```

I'll update it to "Sayyara"

**Review by TimChoy — APPROVED** [review]

File: docs/Design/SoftArchitecture/MG.tex (hunk 1 above)
Comment (arkinmodi) [reply docs/Design/SoftArchitecture/MG.tex]: I think you are looking at an older version. Made the change to our program name in  https://github.com/arkinmodi/project-sayyara/pull/397/commits/49e52ae0f910319509f0bd782a10e21aa7d3c2f7.

Comment (TimChoy) [reply docs/Design/SoftArchitecture/MG.tex]: Yeah it carried forward when I approved the changes, not sure why

**Pull request merged**